app/models/npq_application.rb

```diff
@@ -110,8 +110,8 @@ def in_uk_catchment_area?
     UK_CATCHMENT_AREA.include?(teacher_catchment)
   end
 
-  def self.participant_declaration_finder(participant_identity_id)
-    ParticipantDeclaration::NPQ.find_by_participant_profile_id(ParticipantProfile.find_by_participant_identity_id(participant_identity_id)&.id)
+  def self.completed_participant_declaration_finder(participant_identity_id)
+    ParticipantDeclaration::NPQ.billable_or_changeable.for_declaration("completed").find_by(participant_profile_id: ParticipantProfile.find_by_participant_identity_id(participant_identity_id)&.id)
```

Comment:
Let's chat about this tomorrow, I just realised we aren't scoping the profile to an NPQ profile which could be a big issue, and the service calling this is quite hard to follow.
I would change this whole lookup to something like the following:
```
completed_declaration = npq_application.profile.participant_declarations.npq.complete.billable_or_changeable.order(created_at: :desc).first
```

and we get the outcome for it in the other service:
```
completed_declaration.outcomes.latest.state
```
But yea let's chat tomorrow :)